### PR TITLE
Fixes for prow-performance  bootstrap and deployment jobs.

### PR DESCRIPTION
**Pull request by fgimenez**
/cc @marceloamaral @dhiller 

Signed-off-by: Federico Gimenez <fgimenez@redhat.com>

### CODE REVIEW

File: github/ci/prow-performance-workloads/hack/common.sh [hunk 1]
```diff
@@ -12,7 +12,14 @@ setup(){
     mkdir -p ~/.ssh && chmod 0700 ~/.ssh
     extract_secret 'prowPerformance.sshKey' ~/.ssh/id_rsa
     chmod 0600 ~/.ssh/id_rsa
+    printf "\n" >> ~/.ssh/id_rsa
```
Comment (marceloamaral) [inline github/ci/prow-performance-workloads/hack/common.sh:15]: It was probably for debugging, but I don't think it's a good idea to print this out and make private key public....

Comment (fgimenez) [reply github/ci/prow-performance-workloads/hack/common.sh:15]: @marceloamaral This command won't show the private key file contents, just makes sure that it always has a new line at the end

Comment (marceloamaral) [reply github/ci/prow-performance-workloads/hack/common.sh:15]: ohh that's right, I didn't see it very well!

**Comment by fgimenez** [comment]
Added changes to setup DNS in the control plane nodes, successful execution here https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/post-project-infra-prow-performance-workloads-bootstrap-nodes/1463176890721243136

**Review by dhiller — APPROVED** [review]
/approve

**Comment by dhiller** [comment]
I was a bit confused regarding the file extension changes from four to three letters, was there a reason for this?

**Comment by fgimenez** [comment]
/retest

error not related to the changes in this PR

**Pull request merged**